presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -415,12 +419,15 @@ public static LocalDateTime fromPrestoTimestamp(long value)
 
             case Types.NUMERIC:
             case Types.DECIMAL:
-                int decimalDigits = type.getDecimalDigits();
+                if (!type.getDecimalDigits().isPresent()) {
+                    return Optional.of(decimalColumnMapping(createDecimalType(Decimals.MAX_PRECISION, getDecimalDefaultScale(session)), getDecimalRoundingMode(session)));
+                }
+                int decimalDigits = type.getDecimalDigits().get();
```

Comment:
This changes are not tested. Please add test to presto-base-jdbc/src/test/java/io/prestosql/plugin/jdbc/TestJdbcIntegrationSmokeTest.java? 

Also it is possible to trigger this code and test it with mysql or sqlserver?

Reply:
> Also it is possible to trigger this code and test it with mysql or sqlserver?

It's not relevant for Mysql and Sqlserver since they don't have such variable decimals - using just `decimal` means a scale of `0` with a default precision of `10`(Mysql) and `18`(Sqlserver).

For Oracle it will be relevant with the `number` data type that is  limited to precision of 38 digits. (all values can be retrieved by at least setting `decimal-rounding-scale=0`)

Reply:
Thanks.